src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post" 
+  >
+      <fieldset>
+        <legend> Personal information </legend>
+        <label class="form-field">
+          Surname:
+          <input type="text"
+            name="Surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
```

Comment:
You need to add spaces between child components to improve readability
```suggestion
        </label>
        
        <label class="form-field">
```